mmcv/ops/points_in_boxes.py

```diff
@@ -18,7 +18,8 @@ def points_in_boxes_part(points, boxes):
             LiDAR/DEPTH coordinate, (x, y, z) is the bottom center
 
     Returns:
-        box_idxs_of_pts (torch.Tensor): (B, M), default background = -1
+        torch.Tensor: Return the box indies of points with the shape of
+        (B, M, T). Default background = 0.
```

Comment:
Return the box indies of points with the shape of (B, M). Default background = -1.